clients/drcachesim/tracer/tracer.cpp

```diff
@@ -201,6 +201,87 @@ drmemtrace_custom_module_data(void * (*load_cb)(module_data_t *module),
         return DRMEMTRACE_ERROR;
 }
 
+/* sideline */
+static drvector_t buf_queue;
+static uint queue_size = 0;
+static uint queue_size_mask = 0;
```

Comment:
Hmm, no, if this is based on queue_size, then queue_size must be the length of the vector?  Why do we need a separate var for that: it's already in buf_queue, right?